src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_PaymentOptionsScreen.tsx

```diff
@@ -84,25 +127,31 @@ export const Profile_PaymentOptionsScreen = ({}: PaymentOptionsProps) => {
           leftButton={{type: 'back'}}
         />
       </View>
-      <View style={style.contentContainer}>
-        <FlatList
-          ListHeaderComponent={
-            !isLoading && showError ? (
-              <GenericError />
-            ) : !isLoading && storedCards.length == 0 ? (
-              <NoCardsInfo />
-            ) : null
-          }
-          refreshing={isLoading}
-          onRefresh={() => refreshCards()}
-          progressViewOffset={0}
-          data={storedCards}
-          keyExtractor={(item) => 'card-' + item.id}
-          renderItem={({item}: RecurringPaymentRenderItem) => (
-            <Card card={item} removePaymentHandler={handleRemovePayment} />
-          )}
-        />
-      </View>
+      {showError && <GenericError />}
+      <ScrollView
+        style={style.contentContainer}
+        refreshControl={
+          <RefreshControl refreshing={isLoading} onRefresh={refreshCards} />
+        }
+      >
+        {storedCards.length > 0 && (
+          <Section>
+            {storedCards.map((item) => (
```

Comment:
Just my taste to call each item `storedCard` or `sc` instead of the generic `item` in mappings like this